external/kinesis-asl/src/test/scala/org/apache/spark/streaming/kinesis/KinesisBackedBlockRDDSuite.scala

```diff
@@ -101,6 +103,36 @@ abstract class KinesisBackedBlockRDDTests(aggregateTestData: Boolean)
     }
   }
 
+  testIfEnabled("Basic reading from Kinesis with modified configurations") {
```

Comment:
I don't see how this test actually tests the configuration setting. It just tests if things work, not that the configurations are actually picked up.